react/index.tsx

```diff
@@ -8,7 +8,13 @@ import {
   CartItem,
 } from "./typings/events";
 
-let log = canUseDOM && window.oct8neVtex.enableLogs ? console.log : () => {};
+let log = (msg : any) => {
+    let enableLogs = window.oct8neVtex?.enableLogs;
+    if ( enableLogs === undefined) enableLogs = false;
+    if (enableLogs === false) return;
+    console.log(msg);  
```

Comment:
Is this console.log necessary?

Reply:
It is not necessary. But it will help me debug problems. 
If something goes wrong, I can set window.oct8neVtex.enableLogs = true in the console and see what it is going on (events, and other stuff). I did this because with typescript and all the bundles and minification the code becomes unreadable, and it is difficult to track the code in the browser file explorer.
I can remove it if you want